workflow/usage.rst

```diff
@@ -175,12 +175,63 @@ what actions are allowed on a blog post::
     $transitions = $workflow->getEnabledTransitions($post);
 
 Using Events
-------------
+============
 
 To make your workflows even more powerful you could construct the ``Workflow``
-object with an ``EventDispatcher``. You can now create event listeners to
-block transitions (i.e. depending on the data in the blog post). The following
-events are dispatched:
+object with an ``EventDispatcher``. If you are using the ``Workflow`` component within
+the Symfony standard edition, this is already done for you.
+
+When a workflow subject's state is updated, the component dispatches several events that
+enable you to alter the component's behaviour:
+
+* ``workflow.[workflow name].guard.[transition name]``: dispatched before applying a transition.
```

Comment:
Each time an "event" is dispatched, there are actually 3 events:
* `workflow.guard`
* `workflow.[workflow name].guard`
* `workflow.[workflow name].guard.[transition name]`

cf https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/Workflow.php#L217-L219

The same comments applies to others event too.